lib/model_2_factory/model_2_factory.rb

```diff
@@ -81,12 +82,23 @@ def self.create_factory_dir(dirname)
     FileUtils.mkdir_p(dirname) unless File.directory?(dirname)
   end
 
+  def self.red_colorize(msg)
+    "\e[31m#{msg}\e[0m"
+  end
+
+  def self.warnings_for_dependencies
+    unless Gem.loaded_specs.has_key? 'factory_girl_rails'
```

Comment:
__[rubocop]__
* Use a guard clause instead of wrapping the code inside a conditional expression. [:ref](https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)
* Use `Hash#key?` instead of `Hash#has_key?`. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-key)

<!-- Comment by [SideCI](https://sideci.com) -->